PowerShell/JumpCloud Module/Private/NestedFunctions/Get-JCObject.ps1

```diff
@@ -152,13 +197,24 @@ Function Get-JCObject {
                     }
                     If ($JCType.TypeName.TypeNameSingular -in ('g_suite', 'office_365')) {
                         # Hacky logic to get g_suite and office_365 directories
+
                         If ($ReturnCount -eq $true) {
                             $Directory = $Result.results | Where-Object { $_.Type -eq $JCType.TypeName.TypeNameSingular }
                             $Result.totalCount = ($Directory | Measure-Object).Count
                             $Result.results = $Directory
                         } Else {
                             $Result = $Result | Where-Object { $_.Type -eq $JCType.TypeName.TypeNameSingular }
                         }
+                        # manual filter since the API doesn't support it
+                        foreach ($dirResult in $Result) {
+                            if ($dirResult.Name -eq $SearchByValueItem) {
```

Comment:
Are we all okay with the search being -eq? we could get a bit more specific or less so. 

Right now for Radius Server: name `test` will return results for
"someTest", "test", "a name with test in there somewhere"

For gworkspace/ o365, name `test` will return results for:
objects named exactly "test", "TEST", "tEsT"

Would it make sense to change this work more like search term? my guess is no?